<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Goku" failures="9" tests="17" name="Surefire test" time="31.522" errors="0" timestamp="2024-10-02T18:18:49 IST">
  <testcase classname="com.qa.petstore.test.DataDrivenTests" name="testPostUser" time="4.764">
    <failure type="org.testng.internal.reflect.MethodMatcherException" message="
[public void com.qa.petstore.test.DataDrivenTests.testPostUser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testPostUser([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2, java.lang.String arg3, java.lang.String arg4, java.lang.String arg5, java.lang.String arg6])
Arguments: [(java.lang.String) 1010,(java.lang.String) testuser1,(java.lang.String) John,(java.lang.String) Candey]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.qa.petstore.test.DataDrivenTests.testPostUser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testPostUser([java.lang.String arg0, java.lang.String arg1, java.lang.String arg2, java.lang.String arg3, java.lang.String arg4, java.lang.String arg5, java.lang.String arg6])
Arguments: [(java.lang.String) 1010,(java.lang.String) testuser1,(java.lang.String) John,(java.lang.String) Candey]
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testPostUser -->
  <testcase classname="com.qa.petstore.test.DataDrivenTests" name="testGetUserByName" time="5.253">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.qa.petstore.test.DataDrivenTests.testGetUserByName(DataDrivenTests.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="com.qa.petstore.test.DataDrivenTests" name="testGetUserByName" time="1.186">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.qa.petstore.test.DataDrivenTests.testGetUserByName(DataDrivenTests.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="com.qa.petstore.test.DataDrivenTests" name="testGetUserByName" time="1.126">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.qa.petstore.test.DataDrivenTests.testGetUserByName(DataDrivenTests.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="com.qa.petstore.test.DataDrivenTests" name="testGetUserByName" time="1.259">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at com.qa.petstore.test.DataDrivenTests.testGetUserByName(DataDrivenTests.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="com.qa.petstore.test.DataDrivenTests" name="testDeleteUserByName" time="1.13">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.qa.petstore.test.DataDrivenTests.testDeleteUserByName(DataDrivenTests.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="com.qa.petstore.test.DataDrivenTests" name="testDeleteUserByName" time="1.109">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.qa.petstore.test.DataDrivenTests.testDeleteUserByName(DataDrivenTests.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="com.qa.petstore.test.DataDrivenTests" name="testDeleteUserByName" time="1.103">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at com.qa.petstore.test.DataDrivenTests.testDeleteUserByName(DataDrivenTests.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="com.qa.petstore.test.DataDrivenTests" name="testDeleteUserByName" time="1.112">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at com.qa.petstore.test.DataDrivenTests.testDeleteUserByName(DataDrivenTests.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="com.qa.petstore.test.User1Tests" name="testPostUser" time="2.027"/>
  <testcase classname="com.qa.petstore.test.User1Tests" name="testGetUserByName" time="1.188"/>
  <testcase classname="com.qa.petstore.test.User1Tests" name="testUpdateUser" time="2.307"/>
  <testcase classname="com.qa.petstore.test.User1Tests" name="testDeleteUserByName" time="1.081"/>
  <testcase classname="com.qa.petstore.test.UserTests" name="testPostUser" time="1.144"/>
  <testcase classname="com.qa.petstore.test.UserTests" name="testGetUserByName" time="1.115"/>
  <testcase classname="com.qa.petstore.test.UserTests" name="testUpdateUser" time="2.241"/>
  <testcase classname="com.qa.petstore.test.UserTests" name="testDeleteUserByName" time="1.105"/>
</testsuite> <!-- Surefire test -->
